kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -663,10 +665,15 @@ redirect
 redirect cmd path = do
     config <- ask
     get >>= runInterpreter config >>= put
-    putStrLn' "File created."
   where
     redirectToFile :: String -> IO ()
-    redirectToFile = writeFile path
+    redirectToFile str = do
+        isCorrectPath <- checkIfCorrectFilePath path
```

Comment:
This could be refactored into something like
```
whenPathIsReachable :: FilePath -> (FilePath -> IO ()) -> IO ()
whenPathIsReachable path action =
    ifM
        (checkIfCorrectFilePath path)
        (action path)
        $ putStrLn "Directory does not exist."
```

Then you could replace all the copies with this, I think.